pegasus/sites.v3/code.org/views/header_banner.haml

```diff
@@ -0,0 +1,15 @@
+:ruby
+  banner_titles = {
```

Comment:
As an alternative to this, presumably we could have each page indicate which banner it wants with a parameter inside the `---`s at the top of the page..